recipe/meta.yaml

```
@@ -1,29 +1,29 @@
 {% set name = "torchsde" %}
-{% set version = "0.2.4" %}
+{% set version = "0.2.6" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/google-research/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 8a4f1f7dd3011bf6b3a1a8c97b9e8ab4093da79b863a83e022d00dd677cc0b7d
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 81d074d3504f9d190f1694fb526395afbe4608ee43a88adb1262a639e5b4778b
 
 build:
-  skip: True  # [win and (py<37 or py>38)]
-  number: 3
+  skip: True  # [py<38]
+  number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   build:
+    - python >=3.8
```

Does this need to be in the build deps as well?
```suggestion
    - python
```